src/main/java/org/broadinstitute/consent/http/models/DarDecisionMetrics.java

```diff
@@ -113,6 +120,28 @@ private void setDatasetId(DataSet dataset) {
     }
   }
 
+  private void setCountUniqueUser(DataAccessRequest dar) {
+    List<String> emails = null;
+    if (Objects.nonNull(dar.data)) {
+      ArrayList<Collaborator> collabs = new ArrayList<>();
+      if (Objects.nonNull(dar.data.getInternalCollaborators())) {
+        collabs.addAll(dar.data.getInternalCollaborators());
+      }
+      if (Objects.nonNull(dar.data.getLabCollaborators())) {
+        collabs.addAll(dar.data.getLabCollaborators());
+      }
+      emails = collabs
+        .stream()
+        .filter(Objects::nonNull)
+        .map(Collaborator::getEmail)
+        .filter(Objects::nonNull)
+        .collect(Collectors.toList());
+    }
+    this.countUniqueUser = Objects.nonNull(emails) ? emails.size() : 0;
```

Comment:
There are at least two extraneous holder variables that can be extracted out. `emails` and `collabs` are both unnecessary:
```
    if (Objects.nonNull(dar.getData())) {
      this.countUniqueUser = (int) Streams
          .concat(
              dar.getData().getLabCollaborators().stream(),
              dar.getData().getInternalCollaborators().stream())
          .filter(Objects::nonNull)
          .map(Collaborator::getEmail)
          .filter(Objects::nonNull)
          .map(String::toLowerCase)
          .distinct()
          .count();
    } else {
      this.countUniqueUser = 0;
    }
```

That can also be re-written to be a one-liner ternary:
```
    this.countUniqueUser =
        (Objects.nonNull(dar.getData())) ? 
            (int) Streams
              .concat(
                  dar.getData().getLabCollaborators().stream(),
                  dar.getData().getInternalCollaborators().stream())
              .filter(Objects::nonNull)
              .map(Collaborator::getEmail)
              .filter(Objects::nonNull)
              .map(String::toLowerCase)
              .distinct()
              .count()
            : 0;
```